src/entities/swap/swaps/v3/index.ts

```diff
@@ -577,4 +609,10 @@ export class SwapV3 implements SwapBase {
         }
         return swaps;
     }
+
+    private to(): Address {
```

Comment:
Do you think this is actually useful? I find it less explicit.

Reply:
Yeah, I changed because I found myself repeating the code of checking for `isBatchSwap` and assigning `to` to the proper address, but I agree that having these things closer to where it's being used is more explicit.
I'll rollback that change 👍